libs/s25client/s25client.cpp

```diff
@@ -56,6 +55,8 @@
 #include <csignal>
 #endif
 
+namespace bfs = boost::filesystem;
```

Comment:
is bfs used?

Reply:
Yes. I only removed the include that used to define this (want to get rid of the "keep" headers)